stack.yaml

```diff
@@ -0,0 +1,66 @@
+# This file was automatically generated by 'stack init'
```

Comment:
I'm not an expert on stack, so forgive me if this question is ignorant: Is this generated file supposed to be committed?